app/views/overview/_mobile-client-row.html

```diff
@@ -57,13 +57,32 @@ <h3>
               </button>
               <span class="pficon pficon-info"></span>
               {{row.servicesNotBoundCount}} mobile services are not bound to this client. 
-              <a href="" ng-click="row.navigateToMobileServices()">Bind them to use with this client.</a>
+              <a href="" ng-href="{{ row.navigateToMobileServices() }}">Bind them to use with this client.</a>
             </div>
           </div>
           <div class="col-md-4">
             <mobile-client-config mobile-client="row.apiObject"></mobile-client-config>
           </div>
         </div>
+        <div ng-if="(row.builds | size)">
+            <div class="component-label section-label">Mobile Builds</div>
+            <div ng-repeat="build in row.builds">
+              <div class="row">
+                <div class="col-md-6">
+                  <h3>
+                    <a href="" ng-href="{{ row.navigateToBuild(build) }}">{{ build.metadata.labels.buildconfig }}</a>
+                  </h3>
+                </div>
+                <div class="col-md-6">
+                    <span class="build-status-icon" ng-if="build.status.phase.toLowerCase() === 'running'"><i class="fa fa-refresh"></i></span>
```

Comment:
Can we use the exact html used in the pipelines page I think its in `views/directives/_build-pipeline-expanded.html`